src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -49,6 +49,8 @@ export class IssueReporter extends Disposable {
 	private receivedPerformanceInfo = false;
 	private shouldQueueSearch = false;
 	private hasBeenSubmitted = false;
+	private openReporter = false;
```

Comment:
Does this signal to open a reporter or when a reporter is open?

Reply:
this signals when a reporter is opened

Reply:
isn't this running in an issue reporter so wouldn't it always be open? Maybe I'm not understanding what this property relates to